src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball ball1 = Lottery.getRandomBall();
+        Ball ball2 = Lottery.getRandomBall();
+        Ball ball3 = Lottery.getRandomBall();
+
+        System.out.println(ball1);
+        System.out.println(ball2);
+        System.out.println(ball3);
     }
```

Comment:
Avoid creating multiple instances of the same class when one instance can be reused. In this case, you're calling `Lottery.getRandomBall()` which suggests that `Lottery` may be unnecessarily instantiated multiple times. Consider creating a single instance of `Lottery` and reusing it.